### PR TITLE
Add ACK Section

### PR DESCRIPTION
**Pull request by nopara73**

### CODE REVIEW

File: main.tex [hunk 1]
```diff
@@ -531,6 +531,12 @@ \subsection{Perfect Hiding}
 
 To unconditionally preserve user privacy in the event that the hardness assumption of the discrete logarithm problem in $\mathbb{G}$ is broken we can add an additional randomness term $r_i^{\prime}$ used with an additional generator $G_h^{\prime}$ to the amount commitments $M_{a_i}$, and similarly another randomness term $z_i^{\prime}$ and generators $G_a^{\prime}, G_{x_0}^{\prime}, G_{x_1}^{\prime}, G_V^{\prime}$ in order to obtain unconditional unlinkability for the commitments.\footnote{Assuming the coordinator is not able to attack network level privacy and the proofs of knowledge are unconditionally hiding.}
 
+\section{Acknowledgements}
+
+We would like to acknowledge the inputs and invaluable contributions of ZmnSCPxj, Yahia Chiheb, Thaddeus Dryja, Adam Gibson, Dan Gould, Ethan Heilman, Max Hillebrand, Aviv Milner, Jonas Nick, Lucas Ontivero, Tim Ruffing, Ruben Somsen and Greg Zaverucha to this paper.
+
+Bitcoin transactions
```
Comment (nopara73) [inline main.tex:538]:
```suggestion
```

File: main.tex [hunk 2]
```diff
@@ -531,6 +531,11 @@ \subsection{Perfect Hiding}
 
 To unconditionally preserve user privacy in the event that the hardness assumption of the discrete logarithm problem in $\mathbb{G}$ is broken we can add an additional randomness term $r_i^{\prime}$ used with an additional generator $G_h^{\prime}$ to the amount commitments $M_{a_i}$, and similarly another randomness term $z_i^{\prime}$ and generators $G_a^{\prime}, G_{x_0}^{\prime}, G_{x_1}^{\prime}, G_V^{\prime}$ in order to obtain unconditional unlinkability for the commitments.\footnote{Assuming the coordinator is not able to attack network level privacy and the proofs of knowledge are unconditionally hiding.}
 
+\section{Acknowledgements}
+
+We would like to acknowledge the inputs and invaluable contributions of ZmnSCPxj, Yahia Chiheb, Thaddeus Dryja, Adam Gibson, Dan Gould, Ethan Heilman, Max Hillebrand, Aviv Milner, Jonas Nick, Lucas Ontivero, Tim Ruffing, Ruben Somsen and Greg Zaverucha to this paper.
+
+
```
Comment (nopara73) [inline main.tex:538]:
```suggestion
```

File: main.tex [hunk 3]
```diff
@@ -531,6 +531,10 @@ \subsection{Perfect Hiding}
 
 To unconditionally preserve user privacy in the event that the hardness assumption of the discrete logarithm problem in $\mathbb{G}$ is broken we can add an additional randomness term $r_i^{\prime}$ used with an additional generator $G_h^{\prime}$ to the amount commitments $M_{a_i}$, and similarly another randomness term $z_i^{\prime}$ and generators $G_a^{\prime}, G_{x_0}^{\prime}, G_{x_1}^{\prime}, G_V^{\prime}$ in order to obtain unconditional unlinkability for the commitments.\footnote{Assuming the coordinator is not able to attack network level privacy and the proofs of knowledge are unconditionally hiding.}
 
+\section{Acknowledgements}
+
+We would like to acknowledge the inputs and invaluable contributions of ZmnSCPxj, Yahia Chiheb, Thaddeus Dryja, Adam Gibson, Dan Gould, Ethan Heilman, Max Hillebrand, Aviv Milner, Jonas Nick, Lucas Ontivero, Tim Ruffing, Ruben Somsen and Greg Zaverucha to this paper.
```
Comment (MaxHillebrand) [inline main.tex:536]: Aren't these usually ordered alphabetically?

Comment (nopara73) [reply main.tex:536]: No, I thought that, too, but when I wanted to change the order of the author list, @nothingmuch pointed out in a call that it's usually alphabetical by last name.

Comment (nothingmuch) [reply main.tex:536]: that was about authors - there's two conventions, either alphabetical by last name for authors, or primary author first, i have no clue if there's conventions about acknowledgements, but i think ordering by alphabetical name makes a lot of sense in this case

**Pull request merged**